handlers/api.go

```diff
@@ -1704,6 +1707,12 @@ func ApiValidatorIncomeDetailsHistory(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	maxValidators := getUserPremium(r).MaxValidators
 
+	latestEpoch, limit, err := getBalanceHistoryQueryParameters(r.URL.Query())
```

Comment:
I assume the plan here was to use the new `getIncomeDetailsHistoryQueryParameters`?